python/triton/runtime/interpreter.py

```diff
@@ -1190,7 +1195,7 @@ def _compile_and_exec(self, transformed_ast):
         for key, value in globals().items():
             if key not in fn_globals:
                 fn_globals[key] = value
-        exec(compiled_code, fn_globals, local_namespace)
+        exec(compiled_code, fn_globals, local_namespace)  # nosec B102
```

Comment:
Using `exec` is considered unsafe, but I'm not sure yet how to rewrite this place, additional input data checks may be required. However, at least this way the unsafe place is highlighted.